docs/sources/mimir/configure/configure-tsdb-block-upload.md

```diff
@@ -55,10 +55,12 @@ We intend to add validation of uploaded blocks in a future release, which would
 
 ### The results-cache needs flushing
 
-After uploading one or more blocks, the results-cache needs flushing. The reason is that Grafana Mimir caches query results
-for queries that don’t touch the most recent 10 minutes of data. After uploading blocks however, queries may return different
-results (because new data was uploaded). Therefore cached results may be wrong, meaning the cache should manually be flushed
-after uploading blocks.
+Grafana Mimir caches query results for queries that don't require the most recent 10 minutes of data.
```

Comment:
This is not entirely correct. Queries requirement the last 10m are still cached, but we don't cache the last 10m samples. I suggest to improve it.

Reply:
WDYT:

> Grafana Mimir caches query results that are 10 minutes old.

(This is just introduction sentence to explain the problem with caching and backfill)

Reply:
WDYT about:

> Grafana Mimir caches samples older than 10 minute (configurable via `-query-frontend.max-cache-freshness`) in the range query results.

Now, that 10m is even wrong. The default is 1m 😱 . Going to open a PR to change the default in Mimir, cause I think we've always communicated 10m to everyone.

Reply:
I still see default being 10m :thinking: